### PR TITLE
Changed tests of torch.tensordot instead of adding min_value and max_…

### PR DESCRIPTION
**Pull request by paulaehab**
…value to 5 and -5 to skip this error < Invalid dimension to be accessed. Now only supports access to dimension 0 to 9 for paddle framework> I add assume to skip tests when dim >9  for paddle framework
Close

### CODE REVIEW

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

**Comment by paulaehab** [comment]
Close #9856
